spec/models/person_spec.rb

```diff
@@ -1,10 +1,34 @@
 require 'rails_helper'
 
-describe Person, type: :model do
-  subject { FactoryGirl.create(:person) }
-
-  describe 'creates a people' do
-    it { is_expected.not_to be_new_record }
-    it { is_expected.to be_valid }
+describe Person do
+  it 'is valid with a name and height' do
+    person = Person.new(
+        name: 'Joop ter Heul',
+        height: 1.80 )
+    expect(person).to be_valid
+  end
+  it "is invalid without a name" do
+    person = Person.new(name: nil)
+    person.valid?
+    expect(person.errors[:name]).to include("can't be blank")
```

Comment:
This is the correct way of testing validations. `person` has a specific error on name.